### PR TITLE
adding inheritance from bacterium class

### PR DESCRIPTION
**Pull request by Ati-74**
please review two things: 1) adding inheritance from bacterium class 2)test file modification. thanks
@cheekolegend

### CODE REVIEW

**Review by cheekolegend — CHANGES_REQUESTED** [review]
Physical contact bacterium should only have functionalities associated with physical contact. Remove variables associated with toxin fields.

File: src/bsim/winter2021/Bacterium.java [hunk 1]
```diff
@@ -15,6 +15,8 @@
     public final long origin_id;
     public long parent_id;
     public int lifetime;
+    public int toxintime;   
+    public int contact;
```
Comment (cheekolegend) [inline src/bsim/winter2021/Bacterium.java]: Remove these from Bacterium.java; these should only be associated with certain types of bacteria

File: src/bsim/winter2021/Bacterium.java [hunk 2]
```diff
@@ -51,6 +53,8 @@ public Bacterium(BSim sim, Vector3d px1, Vector3d px2){
         this.origin_id = id;
         this.parent_id = -1;
         lifetime = 0;
+        toxintime=0;
+        contact=0;
```
Comment (cheekolegend) [inline src/bsim/winter2021/Bacterium.java]: Remove these from Bacterium.java; these should only be associated with certain types of bacteria

File: src/bsim/winter2021/Bacterium.java [hunk 3]
```diff
@@ -69,6 +73,8 @@ public Bacterium(BSim sim, Vector3d px1, Vector3d px2, long origin_id, long pare
         this.origin_id = origin_id;
         this.parent_id = parent_id;
         lifetime = 0;
+        toxintime=0;
+        contact=0;
```
Comment (cheekolegend) [inline src/bsim/winter2021/Bacterium.java]: Remove these from Bacterium.java; these should only be associated with certain types of bacteria

File: src/bsim/winter2021/Bacterium.java [hunk 4]
```diff
@@ -218,6 +224,8 @@ public Bacterium divide() {
 
         this.parent_id = this.id;
         this.lifetime = 0;
+        this.toxintime = 0;      
+        this.contact= 0;
```
Comment (cheekolegend) [inline src/bsim/winter2021/Bacterium.java]: Remove these from Bacterium.java; these should only be associated with certain types of bacteria

**Review by cheekolegend — CHANGES_REQUESTED** [review]

**Comment by Ati-74** [comment]
Changes associated with Bacterium.java are implemented, and the test file is updated.
@cheekolegend

**Review by cheekolegend — APPROVED** [review]

**Pull request merged**